pkg/server/setup.go

```diff
@@ -118,6 +119,7 @@ func setupCli(c *gin.Context) error {
 	if err := crypto.CreateKeyFingerprintFile(); err != nil {
 		return err
 	}
+	_ = wkspace.DownloadReadme()
```

Comment:
why did you hide the error?

Reply:
it's on a critical path and I don't want init to fail if it fails (eg the user still sets up but maybe doesn't have a readme in their repo, no big deal)